ham/utils/constants.py

```diff
@@ -0,0 +1,18 @@
+import os
+
+# PATH constants
+HAM_PATH = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
+SYMLINK_NAME = os.path.join(HAM_PATH, '..', 'current')
```

Comment:
We can use the current directory for this purpose.
But when using outside the root directory of the source there might be some bad side effects on : 

* the `patch` task
* the `bench` task

I've the feelling that it's better to reimplement those tasks to conform with the pip installation instead of modifying them.

For the former we could maybe better rely on the custom node config directory introduced in kolla 3 : http://docs.openstack.org/developer/kolla/advanced-configuration.html#openstack-service-configuration-in-kolla
But we can't get rid of other customization (non openstack / non ini config files) -> But maybe we can offer this feature for user installing from source only (at least in a 1st version)